src/libcore/time.rs

```diff
@@ -263,6 +263,23 @@ impl Duration {
     #[inline]
     pub fn subsec_nanos(&self) -> u32 { self.nanos }
 
+    /// Returns the total number of nanoseconds contained by this `Duration`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(duration_nanos)]
+    /// use std::time::Duration;
+    ///
+    /// let duration = Duration::new(5, 730023852);
+    /// assert_eq!(duration.as_nanos(), 5730023852);
+    /// ```
+    #[unstable(feature = "duration_nanos", issue = "50167")]
```

Comment:
The tracking issue number should be a new issue, not this PR.

Reply:
Done.